domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -365,14 +365,20 @@ data class Behandlingsinformasjon(
         )
     }
 
-    /** Midlertidig migreringsfunksjon fra Behandlingsinformasjon + Grunnlag.Bosituasjon -> Behandlingsinformasjon */
+    /**
+     * Midlertidig migreringsfunksjon fra Behandlingsinformasjon + Grunnlag.Bosituasjon -> Behandlingsinformasjon
```

Comment:
I teorien burde vi vel kunne "bygge ned" behandlingsinformasjon med en og en ting etterhvert som data er migrert og både søknadsbehandling og revurdering benytter nytt format? Hva er årsaken til at vi trenger å migrere disse dataene "tilbake" til behandlingsinformasjon? Er det for å redusere scope for brev+++?

Reply:
Vi har fremdeles Behandlingsinformasjon på vedtaket også tror jeg vi bruker det i litt forskjellige scopes fremdeles. Jeg har ikke helt oversikt/kontroll der, så tenkte det tryggeste var en litt roligere overgang. Dessuten har denne PRen blitt for stor.

Men vi har gjort en del opprydning da! Vi har fjernet behandlingsinformasjon fra revurderingjson og tilhørende typer i frontend. I tillegg har vi fjernet ektefelle og bosituasjon fra behandlingsinformason i frontend. Så vi er godt på vei! Vi bør ha en egen PR der vi gjør tilsvarende på uføre